safeeyes/plugins/smartpause/dependency_checker.py

```diff
@@ -20,12 +20,9 @@
 
 
 def validate(plugin_config):
-    command = None
     if Utility.DESKTOP_ENVIRONMENT == "gnome" and Utility.IS_WAYLAND:
         command = "dbus-send"
-    else:
-        command = "xprintidle"
-    if not Utility.command_exist(command):
-        return _("Please install the command-line tool '%s'") % command
-    else:
-        return None
+        if not Utility.command_exist(command):
+            return _("Please install the command-line tool '%s'") % command
+
```

Comment:
Should we try to make the `xcffib` dependency optional, by putting an import check here?
Or keep it simple and not have maybe-dependencies for bundled components?

Reply:
Can Smart Pause plugin work without the `xcffib` library? If it cannot please add a check here. This check will only disable the plugin (not Safe Eyes) if the dependency is missing.